lib/ast_grammar_normalized.re

```diff
@@ -9,7 +9,7 @@ type ident = string;
 [@deriving show { with_path : false} ]
 type atom =
  | SYMBOL(ident) /* codegen: ident */
- | TOKEN
+ | TOKEN(option(ident))
```

Comment:
no, option(string).
They are the same types effectively (one is a synonymous of the other), but they do not denote the same conceptual
thing.
";" is not the same than "assignement_expression"

Reply:
maybe we could keep TOKEN as is but add STRING(string) like in ast_grammar.re